pkg/sloop/webserver/webfiles.go

```diff
@@ -6,33 +6,29 @@ import (
 	"github.com/salesforce/sloop/pkg/sloop/common"
 	"github.com/spf13/afero"
 	"html/template"
-	"path"
-	"strings"
 )
 
 const (
 	prefix      = "webfiles/"
-	errorString = "Webfile %v is invalid.  Must start with %v"
+	errorString = "Webfile %v does not exist in local directory or in binary form."
 )
 
 // go-bindata -o bindata.go webfiles
 // ReadWebfile is a function which finds the webfiles that have been predefined and converted to binary format.
-// sample input : filepath= "webfiles/index.html"
-func readWebfile(filepath string, fs *afero.Afero) ([]byte, error) {
-	if !strings.HasPrefix(filepath, prefix) {
-		return nil, fmt.Errorf(errorString, filepath, prefix)
-	}
-	data, err := fs.ReadFile(filepath)
-	if err == nil {
-		return data, err
-	}
-	files := AssetNames()
-	//if file exists in binary form
-	if common.Contains(files, filepath) {
-		return Asset(filepath)
-	} else {
-		return nil, err
+// sample input : fileName="index.html"
+func readWebfile(fileName string, fs *afero.Afero) ([]byte, error) {
+	//file exists as a physical file
+	data, err := fs.ReadFile(common.GetFilePath(webFilesPath, fileName))
+	if err != nil {
+		//file exists in binary
+		binFileList := AssetNames()
+		binFileName := common.GetFilePath(prefix, fileName)
+		if common.Contains(binFileList, binFileName) {
+			return Asset(binFileName)
```

Comment:
could all of this block of code be condenced to:
```
    return Asset(common.GetFilePath(prefix, fileName))
```
this is after seeing the `Asset()` is checking filenames, and assuming that the error message works (not confusing & contains the asset name)